src/binding/c/lottieanimation_capi.cpp

```diff
@@ -204,8 +204,13 @@ lottie_animation_property_override(Lottie_Animation_S *animation,
         animation->mAnimation->setValue<rlottie::Property::StrokeWidth>(keypath, (float)width);
         break;
     }
+    case LOTTIE_ANIMATION_PROPERTY_TR_POSITION: {
+        double x = va_arg(prop, double);
```

Comment:
We may need to change double argument type to float as input. 
Not this one, that must be an additional patch.

Reply:
i will make another patch.